faithcomesbyhearing/dbp-web
# FCBHDBP-575 display filesets according to set_size_code column

# Description
I have addressed an issue where the API was incorrectly attempting to fetch filesets using a book that was not associated with the fileset. To resolve this, I have updated the logic in dbp-web to validate that fileset requests are made only when the set_size_code value matches the book. Previously, the logic was limited to considering only text plain filesets, but now it has been updated to consider all types of filesets.

I have changed the file where were the logic as you can see in the follwoing line:

[app/utils/getValidPlainFilesetsByBook.js → app/utils/getValidFilesetsByBook.js](https://github.com/faithcomesbyhearing/dbp-web/compare/nodejs18...faithcomesbyhearing:dbp-web:FCBHDBP-575-V2?expand=1#diff-8c1fc3e99f7fb312c45e77e7638efae2b498614dcfe78ce3b2e0c25d8a7abce7)


## Related Issue

https://fullstacklabs.atlassian.net/browse/FCBHDBP-575

## How to Test
- You should go the path: bible/JPNCJV/MAT/1 and it must load the correct page.
- You should go the path: biible/en1esv/mat/1 and it must load the correct page.
- You should go the path: biible/jpncjv/rut/2 and it must load the correct page.
- You should go the path: biible/engweb/mat/2 and it must load the correct page.
